flocker/node/agents/cinder.py

```diff
@@ -600,6 +600,50 @@ def _openstack_auth_from_config(auth_plugin='password', **config):
     return plugin_class(**plugin_kwargs)
 
 
+def _openstack_verify_from_config(
+        verify_peer=True, verify_ca_path=None, **config):
+    """
+    Create an OpenStack session from the given configuration.
+
+    This turns a pair of options (a boolean indicating whether to
+    verify, and a string for the path to the CA bundle) into a
+    requests-style single value.
+
+    If the ``verify_peer`` parameter is False, then no verification of
+    the certificate will occur.  This setting is insecure!  Although the
+    connections will be confidential, there is no authentication of the
+    peer.  We're having a private conversation, but we don't know to
+    whom we are speaking.
+
+    If the ``verify_peer`` parameter is True (the default), then the
```

Comment:
If `verify_peer is False` you can distinguish between:

1. the system is almost working, and it is just the CA setup that is wrong
1. the system is not working for other reasons

Since it is an easy but insecure solution, it might be better to use it mainly for support. Hence, why I have not documented in the installation instructions.  However, I'm willing to document it as well.